modules/ovirt-csi-driver-storage-class.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-ovirt.adoc
+
+[id="ovirt-csi-driver-storage-class_{context}"]
+= oVirt CSI driver storage class
+
+OpenShift Container Platform will create a default `StorageClass` *ovirt-csi-sc* which is used for creating dynamically provisioned persistent volumes (for information on storage classes see link:https://deploy-preview-28872--osdocs.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi.html#csi-dynamic-provisioning_persistent-storage-csi[Dynamic Provisioning]).
+To create additional storage classes for different configurations, create and save a file with the `StorageClass` object described by the following YAML:.
+
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: <STORAGE-CLASS-NAME> [1]
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "false" [2]
```

Comment:
number 2 and 4 is in bold and red color - there is any reason for that?

Reply:
no idea - nothing in the document formatting indicates bold in that section